charts/rancher-webhook/templates/validate-psp-install.yaml

```diff
@@ -0,0 +1,5 @@
+#{{- if .Values.global.cattle.psp.enabled }}
+#{{- if not (.Capabilities.APIVersions.Has "policy/v1beta1/PodSecurityPolicy") }}
+#{{- required "The target cluster does not have the PodSecurityPolicy API resource. Please disable PSPs in this chart before proceeding." "" -}}
```

Comment:
Requirements for this changed to require `fail` instead of `required`. Would you mind changing this to be in line with our other charts?